src/main/webapp/static/js/init.js

```diff
@@ -1,10 +1,31 @@
 (function($){
   $(function(){
-
+    var web3Ready = false;
     $('.button-collapse').sideNav();
     $('#navButton').sideNav();
     
     $('select').material_select();
+    $('#web3').click(function() {
+      if (!web3Ready){
+        if (window.ethereum) {  
+          window.web3 = new Web3(ethereum);  
+          window.ethereum.enable()
+        } else if (window.web3) {
```

Comment:
Does `window.web3` indicate _not_ MetaMask, but another Web3 wallet provider?

Reply:
It adds support for an older version of Metamask!